daemon/list.go

```diff
@@ -668,3 +685,56 @@ func populateImageFilterByParents(ancestorMap map[image.ID]bool, imageID image.I
 		ancestorMap[imageID] = true
 	}
 }
+
+// Events returns events generated by daemon
```

Comment:
The description doesn't seem to match the function (i.e., does it really _return_ events?)